PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/networking/modelcontrollers/Requester.java

```diff
@@ -131,16 +126,16 @@ protected void getAdId() {
 
         UserConsentManager userConsentManager = ManagersResolver.getInstance().getUserConsentManager();
         if (userConsentManager.canAccessDeviceData()) {
-            fetchAdIdInfoTask = AdIdManager.initAdId(context, new AdIdFetchListener() {
+            AdIdManager.fetchAdvertisingId(context, new AdIdFetchListener() {
                 @Override
                 public void adIdFetchCompletion() {
-                    LogUtil.info(TAG, "Advertising id was received");
+                    LogUtil.info(TAG, "Advertising id was loaded from cache");
                     makeAdRequest();
```

Comment:
Since the `Requester` class doesn't manage the task's lifecycle (does not cancel it on destruction). we should be sure that a `makeAdRequest()` won't be called on _destroyed_ object. It can lead to undefined behavior.